### PR TITLE
Adding more benchmarks and removing some others

### PR DESCRIPTION
**Pull request by hombit**
- Remove n_features from benchmark matrix: it doesn't affect performance 
- Replace `n_jobs` parameter with `--n_jobs` cmd option, default is 1
- Add `AADForest.fit_known` and `AADEvaluator.loss_graident` benchmarks
- Add `IsolationForest.feature_signature` benchmark

### CODE REVIEW

File: README.md
Comment (matwey) [inline README.md:1]: It is near impossible to understand what has changed here. Could you please put style changes to the separate commit and describe them accordingly?

Comment (hombit) [reply README.md:1]: No, because you people refuse using formatters

Comment (matwey) [reply README.md:1]: It is not a reason to put everything in single commit. I even cannot understand if you updated the README at all.
